Formula/inspectrum.rb

```diff
@@ -0,0 +1,22 @@
+class Inspectrum < Formula
+  desc "Offline radio signal analyser"
+  homepage "https://github.com/miek/inspectrum"
+  url "https://github.com/miek/inspectrum/archive/v0.2.tar.gz"
+  sha256 "50b7db9b86208f414c387700a358eb58364094f3e8a4985f586f4f815645898a"
+  head "http://github.com/miek/inspectrum.git"
```

Comment:
`https://`

Reply:
I'll update that when I merge (although it's weird audit didn't flag it)

Reply:
We also don't audit to check that GitHub `head` URLs end in `.git` so. There are gaps that could be filled, for sure.